lib/guardian/post_revision_guardian.rb

```diff
@@ -13,6 +13,10 @@ def can_hide_post_revision?(post_revision)
     is_staff?
   end
 
+  def can_purge_post_revisions?
+    is_staff? && SiteSetting.can_permanently_delete_revisions?
```

Comment:
Hmm, I was thinking of reusing the existing `can_permanently_delete` site setting here. It controls deleting posts/topics currently, and it would make sense for revisions to be part of the same scope.